package/MDAnalysis/lib/formats/libdcd.pyx

```diff
@@ -344,7 +344,10 @@ cdef class DCDFile:
         cdef int with_unitcell = 1
 
         if isinstance(remarks, six.string_types):
-            remarks = bytearray(remarks, 'ascii')
+            try:
+                remarks = bytearray(remarks, 'ascii')
+            except UnicodeDecodeError:
+                remarks = bytearray(remarks)
```

Comment:
We do try, that's why it is in a *try* loop, but unit test failure drove me to make the fallback for some cases where the encoding causes some kind of issue.